tests/printing/i10693.check

```diff
@@ -0,0 +1,23 @@
+[[syntax trees at end of                     typer]] // tests/printing/i10693.scala
+package <empty> {
+  final lazy module val i10693$package: i10693$package = new i10693$package()
+  final module class i10693$package() extends Object() { 
+    this: i10693$package.type =>
+    def test[A >: Nothing <: Any, B >: Nothing <: Any](a: A, b: B): A | B = a
+    val v0: String | Int = test[String, Int]("string", 1)
+    val v1: Int | String = test[Int, String](1, "string")
+    val v2: String | Int = test[(String | Int), (Int | String)](v0, v1)
```

Comment:
I think that the actual output doesn't have the parentheses here.

Reply:
:heavy_check_mark: